src/common/libsubprocess/client.c

```diff
@@ -38,6 +38,7 @@ struct rexec_response {
     pid_t pid;
     int status;
```

Comment:
Commit message suggestion: describe details of the fbuf behavior change in its own paragraph.